stl/inc/yvals_core.h

```diff
@@ -1204,7 +1205,6 @@
 #define __cpp_lib_memory_resource                   201603L
 #define __cpp_lib_node_extract                      201606L
 #define __cpp_lib_not_fn                            201603L
-#define __cpp_lib_optional                          201606L

```

Comment:
That is not correct, in C++17 we should `#define __cpp_lib_optional                          201606L`

Reply:
Disregard, overlooked that the other thing is also conditional